Pylint failure centrally but green locally

```
datasrc/network.py:1:0: E0611: No name 'NetMessageEx' in module 'datatypes' (no-name-in-module)
datasrc/network.py:1:0: E0611: No name 'NetObjectEx' in module 'datatypes' (no-name-in-module)
datasrc/network.py:1:0: E0611: No name 'NetStringHalfStrict' in module 'datatypes' (no-name-in-module)
datasrc/network.py:360:1: E1123: Unexpected keyword argument 'teehistorian' in constructor call (unexpected-keyword-arg)
datasrc/network.py:399:1: E1123: Unexpected keyword argument 'teehistorian' in constructor call (unexpected-keyword-arg)
datasrc/network.py:403:1: E1123: Unexpected keyword argument 'teehistorian' in constructor call (unexpected-keyword-arg)
```
Locally I have Pylint 2.6.0 vs 2.4.4, failure seen here: https://github.com/ddnet/ddnet/pull/3506/checks?check_run_id=1696328976

--- a/datasrc/network.py
+++ b/datasrc/network.py
@@ -1,3 +1,6 @@
+# pylint: skip-file
+# See https://github.com/ddnet/ddnet/issues/3507
+
 from datatypes import Enum, Flags, NetBool, NetEvent, NetIntAny, NetIntRange, NetMessage, NetMessageEx, NetObject, NetObjectEx, NetString, NetStringHalfStrict, NetStringStrict, NetTick
 
 Emotes = ["NORMAL", "PAIN", "HAPPY", "SURPRISE", "ANGRY", "BLINK"]

